pkg/virt-controller/watch/migration_test.go

```diff
@@ -229,6 +236,10 @@ var _ = Describe("Migration watcher", func() {
 		mockQueue.Wait()
 	}
 
+	addNode := func(node *k8sv1.Node) {
+		nodeSource.Add(node)
```

Comment:
As you've seen this is flaky because it needs more time to sync. instead, you can replace  this with something like `nodeInformer.GetIndexer().Add(node)`